sonic-pcied/scripts/pcied

```diff
@@ -134,7 +130,7 @@ class DaemonPcied(daemon_base.DaemonBase):
         self.aer_stats = {}
         if Id is not None:
             self.device_table.set(self.device_name, [('id', Id)])
-            self.aer_stats = _wrapper_get_pcie_aer_stats(bus=Bus, dev=Dev, func=Fn)
+            self.aer_stats = platform_pcieutil.get_pcie_aer_stats(bus=Bus, device=Dev, func=Fn)
```

Comment:
Why change from `dev` to `device`? It looks like the parameter is called `dev`: https://github.com/Azure/sonic-platform-common/blob/master/sonic_platform_base/sonic_pcie/pcie_base.py#L44

Reply:
It is observed that the parameter name is `dev` in the base class method defined in `pcie_base.py`, while it is overridden as `device` by the derived class method in `pcie_common.py` used by pcied (https://github.com/Azure/sonic-platform-common/blob/master/sonic_platform_base/sonic_pcie/pcie_common.py#L104) unifying them both would resolve the issue

Reply:
Updated the pcie_common.py with this PR : https://github.com/Azure/sonic-platform-common/pull/197
And revert the change in pcied.